component/NavBar-Vertical/styles.css

```diff
@@ -34,20 +32,17 @@
     width: 200px;
```

Comment:
changes: use rem for units more than 16px.

Reply:
Done